prow.sh

```diff
@@ -1253,8 +1256,9 @@ main () {
             focus="($focus|CSI.mock.volume)"
         fi
 
-        if tests_need_non_alpha_cluster; then
-            start_cluster || die "starting the non-alpha cluster failed"
+        if tests_need_non_alpha_cluster && [ "${CSI_PROW_E2E_GATES_LATEST}" ]; then
```

Comment:
This currently skips all non-alpha tests because the variable is empty...

```suggestion
        if tests_need_non_alpha_cluster; then
```